test/shmemx/c11_test_shmemx_test_vector.c

```diff
@@ -0,0 +1,180 @@
+#include <shmem.h>
+#include <shmemx.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+#if defined(__STDC_VERSION__) && __STDC_VERSION__ >= 201112L
+
+#define TEST_SHMEM_TEST_ALL(TYPE)                                                                                    \
+    do {                                                                                                             \
+                                                                                                                     \
+        const int my_pe = shmem_my_pe();                                                                             \
+        const int npes = shmem_n_pes();                                                                              \
+        TYPE *ivars = shmem_calloc(npes, sizeof(TYPE));                                                              \
+        int *status = calloc(npes, sizeof(int));                                                                     \
+        TYPE *cmp_values = malloc(npes *sizeof(TYPE));                                                               \
```

Comment:
Minor nit pick: we (SOS/spec) typically put a space before and after the `*` in something like a multiplication.